scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -49,11 +49,11 @@ fi
 WORKSPACE=/root/project
 
 # Increase nodejs stack size
-if [ -e ~/.emscripten ]
+if ! [ -e /emsdk_portable/node/bin/node_orig ]
 then
-    sed -i -e 's/NODE_JS="nodejs"/NODE_JS=["nodejs", "--stack_size=8192"]/' ~/.emscripten
-else
-    echo 'NODE_JS=["nodejs", "--stack_size=8192"]' > ~/.emscripten
+  mv /emsdk_portable/node/bin/node /emsdk_portable/node/bin/node_orig
+  echo -e '#!/bin/sh\nexec /emsdk_portable/node/bin/node_orig --stack-size=8192 $@' > /emsdk_portable/node/bin/node
```

Comment:
Do we really need bash? ;)

Reply:
As a matter of fact I think we do for having well specified behavior with the arguments here :-).